sphinx/util/docutils.py

```diff
@@ -344,6 +345,32 @@ def config(self):
         return self.env.config
 
 
+class SphinxTranslator(nodes.NodeVisitor):
```

Comment:
Is there any appropriate place for this class? `util` package is not best to me, but I don't understand where is better.